CCSocialShare/android/java/src/org/kawaz/socialshare/SocialShare.java

```diff
@@ -89,4 +94,17 @@ static public void copyFile(File src, File dst) throws Exception {
         in.close();
         out.close();
     }
+
+    static public void onActivityResult(int requestCode, int resultCode, Intent intent) {
+        if (intent == null) {
+            callCallback(2);
```

Comment:
I think it is not comfortable to be used hard coded values.
it is better to define `PostResult` type into another file and pass raw value of them.

```java
public enum PostResult {
    SUCCEED,
    FAILURE,
    CANCELLED
}
```

```java
callCallback(PostResult::SUCCEED.ordinal());
```